### PR TITLE
Add partner data sources

### PR DESCRIPTION
**Pull request by aswanson-nr**
# Summary

Adds data sources for many of our partners.

### CODE REVIEW

File: data-sources/cloudflare/config.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  Cloudflare is on a mission to help build a better Internet. The Cloudflare quickstart will enable you to monitor and analyze web traffic metrics on a dashboard, integrating with New Relic’s database to provide an at-a-glance overview of the most important logs and metrics from your websites and applications.
+
+  For more information check out our website at [cloudflare.com](https://www.cloudflare.com/)
+icon: logo.png
+install:
+  primary:
+    nerdlet:
+      nerdletId: nr1-install-newrelic.quickstart-installation-plan
+      nerdletState:
+        quickstart_id: fc2bb0ac-6622-43c6-8c1f-6a4c26ab5434
```
Comment (jerelmiller) [inline data-sources/cloudflare/config.yml:19]: Can we double check this is the right nerdlet state value? I thought this was camelCase, but I could be wrong.

Comment (aswanson-nr) [reply data-sources/cloudflare/config.yml:19]: @jerelmiller You are correct! It looks like camelCase is the way to go, I'll get these updated.

Comment (aswanson-nr) [reply data-sources/cloudflare/config.yml]: All data sources should be updated to use camelCase

**Review by jerelmiller — APPROVED** [review]
This looks great! So stoked to see this in the product soon!

**Pull request merged**